spark3-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestRollbackToTimestampProcedure.java

```diff
@@ -184,8 +184,8 @@ public void testRollbackToTimestampWithQuotedIdentifiers() {
     String quotedNamespace = quotedNamespaceBuilder.toString();
 
     List<Object[]> output = sql(
-        "CALL %s.system.rollback_to_timestamp('%s', '`%s`', TIMESTAMP '%s')",
-        catalogName, quotedNamespace, tableIdent.name(), firstSnapshotTimestamp);
+        "CALL %s.system.rollback_to_timestamp('%s', TIMESTAMP '%s')",
+            catalogName, quotedNamespace + ".`" + tableIdent.name() + "`", firstSnapshotTimestamp);
```

Comment:
Nit: whitespace change?

Reply:
got it, bad copy from the other rollback method